src/Configuration.php

```diff
@@ -95,6 +156,9 @@ public function setConfiguration($configuration): void
         if (is_array($configuration)) {
             $configuration = json_decode(json_encode($configuration), true);
         }
+        if (empty($this->getName()) == true) {
```

Comment:
```suggestion
        if (empty($this->getName())) {
```
It's the same thing :)

Reply:
Thanks for the suggestion and true- it is a bit of habit I developed over the years :)